File: algoritmos/a002-salario-hora/solver.cpp
#include <bits/stdc++.h>

using namespace std;
 

double solve(int salario_anual) {
    double salario_mensual = salario_anual / 12.0;
    double salario_dia = salario_mensual / 20.0;
    double salario_hora = salario_dia / 8.0;
    return salario_hora;
}


int main() {
    int t;
    cin >> t;
    cout << fixed << setprecision(2);

    while (t--) {
        int n;
        cin >> n;

        double salario_hora = solve(n);

        cout << salario_hora << "\n";
    }
}

File: algoritmos/a002-salario-hora/solver.py
def solve(salario_anual):
    salario_mensual = salario_anual / 12
    salario_dia = salario_mensual / 20
    salario_hora = salario_dia / 8
    return salario_hora


def main():
    t = int(input())
    for _ in range(t):
        salario_anual = int(input())
        salario_hora = solve(salario_anual)
        print(f"{salario_hora:.2f}")


if __name__ == "__main__":
    main()
